crates/solidity/testing/sourcify/src/main.rs

```diff
@@ -1,3 +1,8 @@
+// NOTE: there's a [bug](https://github.com/rust-lang/rust-clippy/issues/11916)
```

Comment:
What about renaming `sourcify.rs` to `sourcify_types.rs` to avoid this issue?


If not, I suggest using a `TODO` comment instead of `NOTE`, so we can grep for it during the upgrade.

Reply:
I changed the comment to `TODO` for now. I was thinking that given that the `sourcify.rs` module is already big enough it would make sense to split it, but it didn't seem relevant to include it in this PR.